stablepay-sdk/src/styles/main.css

```
@@ -1,38 +1,43 @@
-  height: 104px;
-  background: linear-gradient(90deg, #000000 0%, #f7941d 100%);
-  border: 1px solid #bc5f26;
-  border-radius: 3px;
-  color: #ffffff;
-  font-size: 24px;
-  font-weight: bold;
+  background: #ffffff;
+  border: 1px solid #e6ebf1;
+  border-radius: 12px;
+  color: #1a1f36;
+  font-family: "Inter", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)